package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@chakrasandwich/react-shocked-pikachu",
+    "version": "1.0.2",
+    "description": "Inserts a pikachu ascii art into your react component",
+    "license": "MIT",
+    "repository": "nmestrada",
```

Comment:
I believe your repository should include your username and repo name, e.g., `nmestrada/react-shocked-pikachu`

Reply:
Interesting, mine has all this stuff - maybe because I forked and cloned it from the mintbean one:

```
    "repository": {
        "type": "git",
        "url": "git+https://github.com/[your-username]/2020-04-17-personal-npm-package.git"
    },
```